kafka-connect-ftp/src/main/scala/com/datamountaineer/streamreactor/connect/ftp/source/FtpSourceTask.scala

```diff
@@ -59,17 +59,19 @@ class FtpSourcePoller(cfg: FtpSourceConfig, offsetStorage: OffsetStorageReader)
       monitor2topic.keys.toSeq,
       cfg.timeoutMs(),
       cfg.getProtocol,
-      cfg.getString(FtpSourceConfig.fileFilter)
+      cfg.getString(FtpSourceConfig.fileFilter),
+      cfg.getInt(FtpSourceConfig.MonitorSliceSize)
     ),
       fileConverter
     )
   }
 
   def poll(): Stream[SourceRecord] = {
-    val stream = if (buffer.isEmpty) fetchRecords() else buffer
-    val (head, tail) = stream.splitAt(cfg.maxPollRecords)
-    buffer = tail
-    head
+    //val stream = if (buffer.isEmpty) fetchRecords() else buffer
+    //val (head, tail) = stream.splitAt(cfg.maxPollRecords)
+    //buffer = tail
+    //head
+    fetchRecords()
```

Comment:
why is the commented code not needed any more ?

Reply:
Since we work by slices, why would we need a buffer?

Reply:
but you are right, I can simply let the code as it is.